sc-memory/sc-core/sc_memory_context_manager.h

```diff
@@ -122,4 +122,19 @@ void _sc_memory_context_pend_event(
  */
 void _sc_memory_context_emit_events(sc_memory_context const * ctx);
 
+//! Checks if specified sc-memory context has blocking events block.
+sc_bool _sc_memory_context_are_events_blocking(sc_memory_context const * ctx);
```

Comment:
I think, we need this function in C++ API too

Reply:
It is private method. We can't use it in C API